packages/arb-token-bridge-ui/src/token-bridge-sdk/utils.ts

```diff
@@ -10,3 +11,51 @@ export const getChainIdFromProvider = async (provider: Provider) => {
   const network = await provider.getNetwork()
   return network.chainId
 }
+
+export const getBridgeTransferProperties = async ({
+  sourceChainProvider,
+  destinationChainProvider,
+  sourceChainErc20Address
+}: {
+  sourceChainProvider: Provider
+  destinationChainProvider: Provider
+  sourceChainErc20Address?: string
+}) => {
+  const sourceChainId = await getChainIdFromProvider(sourceChainProvider)
+  const destinationChainId = await getChainIdFromProvider(
+    destinationChainProvider
+  )
+
+  const isParentChainEthereumMainnetOrTestnet =
```

Comment:
At this stage we don't know if source chain is the parent chain yet. Should this be `isSourceChainEthereumMainnetOrTestnet`?

Same for subsequent occurrences.